### ISSUE TITLE
e2ee Disclaimer

### ISSUE THREAD

**Issue by jakewb-b** [vector-im/element-call#1180]
### Your use case

Prior to releasing the livekit version to call.element.io, we require a prominent disclaimer. 

Ideally, I would like a suitable icon - e.g. an open or greyed-out padlock icon, with the text 'Not E2EE'. If possible this would then have a tooltip stating: `Element Call is temporarily not E2EE while we test scalability`.

@rufuskahler any chance you could put something together on this?

**Comment by rufuskahler** [comment]
@jakewb-b – Yes, I can look at this. Is the term 'E2EE' sufficient for users or should we explain this in layman's terms?

**Comment by jakewb-b** [comment]
Thanks. We could change the tooltip to say end-to-end encrypted rather than e2ee, but generally speaking I think the people who would care will know what it means - and we probably don't want the main text to be too long.

**Comment by americanrefugee** [comment]
@jakewb-b Here's my suggestion for the text:

"Element Call is temporarily not encrypted while we test scalability."

Here's a link to [screens in Figma](https://www.figma.com/file/AjvSw4lI9iGO5aTe4WAl4r/Element-Call?type=design&node-id=3842-74107&mode=design&t=mMbVaDQmKUjHJEV0-4) for reference.

**Comment by jakewb-b** [comment]
@americanrefugee I think we might need to specify 'end to end', because my understanding is that WebRTC media channels are encrypted by default, but the issue is that by introducing the SFU we are breaking the 'end to end' nature of the encryption. So, saying 'Element Call is temporarily not encrypted' isn't strictly accurate. 
@fkwp can you confirm my understanding there?

Aside from that, though, I think the design works great!

**Comment by fkwp** [comment]
Wrong-ish: As discussed we will stick with @americanrefugee original proposal

@americanrefugee As discussed we will stick with @jakewb-b 's original proposal

**Comment by SimonBrandner** [comment]
The previous comment might have been somewhat misleading - we are going to use `Element Call is temporarily not end-to-end encrypted while we test scalability.`
